config/model.conf

```diff
@@ -0,0 +1,12 @@
+
+[request_definition]
+r = obj, act
```

Comment:
This is just an example policy taken from casbin. It will need changes, once i decide on how to assign roles to specific usrs for pods.

Reply:
Is there unit test for this repo now? If not, how can we test this repo is working?